source/deploy/manual-postgres-migration.rst

```diff
@@ -387,7 +387,7 @@ To avoid performance regression on ``Posts`` and ``FileInfo`` table access, foll
 
 .. note::
 
-  If any of the entries in your  ``Posts`` and ``FileInfo`` tables exceed the limit mentioned above, index creation query will warn with the ``ERROR:  string is too long for tsvector`` log while trying to create these indexes. This means the content that didn't fit into a ``tsvector`` was ignored. If you still want to index the truncated content, you can use ``substring()`` function on the content while creating the indexes. An example query is given below:
+  If any of the entries in your  ``Posts`` and ``FileInfo`` tables exceed the limit mentioned above, index creation query will warn with the ``ERROR:  string is too long for tsvector`` log while trying to create these indexes. This means the content that didn't fit into a ``tsvector`` was ignored. If you still want to index the truncated content, you can use ``substring()`` function on the content while creating the indexes. An example query is given below. If it continues to fail to create the index with the substring of the content, consider decreasing the value gradually until the index is created successfully.
```

Comment:
Do we need to give any guidance (i.e. limits) on decreasing this value?

Reply:
I don't want to give a specific number because it can also fail, so it's a bit trial and error case as far as I am concerned.